src/System.Private.Xml/src/Resources/Strings.resx

```diff
@@ -3413,7 +3413,7 @@ building and deploying the assemblies with the application.
   </data>
   <data name="HelpUsage" xml:space="preserve">
     <value>
-Usage: dotnet {0} [[--assembly &lt;assembly name&gt;] | [&lt;assembly file location&gt;]] [--type] [--debug]</value>
+Usage: dotnet {0} [[--assembly &lt;assembly name&gt;] | [&lt;assembly file location&gt;]] [--type &lt;type name&gt;]</value>
```

Comment:
@huanwu Does this usage message look good to you? Do I need to add [--quiet] or other arguments in the message?